app/routes/__index/dns-records/new.tsx

```diff
@@ -47,15 +47,16 @@ export const action = async ({ request }: ActionArgs) => {
 
 export default function NewDnsRecordRoute() {
   const actionData = useActionData();
+  logger.info(actionData);
 
   return (
     <Container maxW="container.xl" ml={[null, null, '10vw']}>
       <Heading as="h1" size="xl" mt="8">
         Create new DNS Record
       </Heading>
       <Text maxW="lg" mb="3" mt="2">
-        Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has
-        been the industry's standard dummy text ever since the 1500s
+        Choose your unique subdomain name. Then enter Type and Name that will be mapped with your
+        domain (i.e. [subdomain].[username].mystudentproject.ca).
```

Comment:
```suggestion
        Choose a unique subdomain name for the Name field. This will be used to build your domain
        (i.e. [subdomain].[username].mystudentproject.ca).
```
Correct me if I am wrong but don't we want to allow duplicate names? If yes, you should remove the `unique` word as well

Reply:
Same for the edit page

Reply:
You are right. Same domain can exist. The same combination of Name, Type, Value can't exist.

Reply:
Don't we also allow same combinations of those (except for CNAME?)

Reply:
We have validation to reject the same combination in `dns-record.server.ts`

```js
if (await doesDnsRecordExist(data)) {
    throw new Error('DNS Record already exists');
  }
```

Reply:
Oh, looking at https://seneca-open-source.slack.com/archives/C04HR4WV5P0/p1680619763741109?thread_ts=1680618251.249069&cid=C04HR4WV5P0, I guess that is the plan for future but not now